src/main/scala/edu/berkeley/ce/rockslicing/SeedJointSelector.scala

```diff
@@ -0,0 +1,243 @@
+package edu.berkeley.ce.rockslicing
+
+import scala.annotation.tailrec
+
+/**
+  * Selects joints from the input joint set that will be used to seed the RDD in order to maintain
+  * approximately balanced loads among processors.
+  */
+object SeedJointSelector {
+
+  /**
+    * Searches though sequence of joint sets to find best seed joints
+    *
+    * @param jointSets Seq of Seq's of Joints representing input joint sets
+    * @param inputVolume Block representing input rock volume
+    * @param numProcessors Number of processors to be used in the analysis
+    * @return Returns a Seq containing the seed joints that best divide the input volume. If unable to find
+    *         satisfactory seed joints, returns None.
+    */
+  @tailrec
+  def searchJointSets(jointSets: Seq[Seq[Joint]], inputVolume: Block,
+                      numProcessors: Int): Option[Seq[Joint]] = {
+    if (jointSets.isEmpty) {
+      println("ERROR: Unable to find seed joints")
```

Comment:
I'm leaning towards throwing an exception here. What do you think?

My sense is that returning an option and then just calling `get` on it isn't the best style. To me, returning an option means that the function could reasonably return `None` or `Some` and the caller will continue normal operation regardless, which isn't true in our case as far as I can tell.